code_share/code_share/views.py

```diff
@@ -0,0 +1,17 @@
+from django.shortcuts import render
+
+
+def page_not_found_view(request):
```

Comment:
In django docs:
`By default, this is 'django.views.defaults.page_not_found'. If you implement a custom view, be sure it returns an HttpResponseNotFound.` 

Link to the docs [here](https://docs.djangoproject.com/en/1.11/ref/urls/#handler404). Same applies to each custom views with their respective response.